docs/setup.md

```diff
@@ -45,6 +41,12 @@ This ensures that you don't get any certificate errors when running gateway and
 FetchError: request to https://idapi.thegulocal.com/auth?format=cookies failed, reason: unable to verify the first certificate\n
 ```
 
+### Environment File
+
+Populate `.env` and `cypress-ete.env` files by using the examples from [`.env.example`](../.env.example) and [`cypress-ete.env.example`](../cypress-ete.env.example.example). These `.env` files should **never** be committed as they store secrets.
```

Comment:
I corrected an issue with this [here](https://github.com/guardian/gateway/pull/1569/files#diff-4ca53116333428d7d34e6d2a9a64069655c8c37d5d36b856320ac1a0c4cb9b23R22), as it doesn't actually link to a file.

Reply:
ah i didn't notice that, i'll bring that into this PR too